scripts/XMLHandler.py

```diff
@@ -31,14 +33,16 @@ def convertDialogData(self, dialogData, intents):
             if not intentData.generateNodes():
                 continue
 
-            normName = toIntentName('soft', None, intent)
             # construct the XML structure for each intent
-            nodeXml = XML.Element('node', name=normName.decode('utf-8'))
+            nodeXml = XML.Element('node', name=self._make_node_id(intent))
+            # normName = toIntentName('soft', None, intent)
+            # nodeXml = XML.Element('node', name=normName)
 
             conditionXml = XML.Element('condition')
-            conditionXml.text = intent.decode('utf-8') if intent.decode('utf-8').startswith(u'#') else u'#' + intent.decode('utf-8')
+            # conditionXml.text = intent.decode('utf-8') if intent.decode('utf-8').startswith(u'#') else u'#' + intent.decode('utf-8')
+            conditionXml.text = intent if intent.startswith(u'#') else u'#' + intent
```

Comment:
@Tereza-Pytelova :-) this is also for you to double check, I presume you did the utf-8 conversions, did you?